plugins/DuskVerb/tools/tuner/dense32_kurtosis_sweep.py

```
@@ -108,14 +108,19 @@ def build_delays(offsets):
 
 
 def render_kurts(delays, tag):
-    d = f"/tmp/d32_{tag}"; shutil.rmtree(d, ignore_errors=True); os.makedirs(d)
-    env = dict(os.environ, DUSKVERB_FDN_DELAYS=",".join(str(x) for x in delays))
-    r = subprocess.run([str(REND), *WET, "--output-dir", d], capture_output=True, env=env)
-    nb = glob.glob(f"{d}/*_noiseburst.wav")
-    if r.returncode != 0 or not nb: return None, None
-    agg = tail_kurt(nb[0], 2000, 14000)
-    sub = {b: tail_kurt(nb[0], *b) for b in SUB}
-    return agg, sub
+    d = f"/tmp/d32_{tag}_{os.getpid()}"
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/d32_"

(S108)